iothub/device/samples/getting started/RoutingTutorial/horizontal-arm/testenvvars_SET.cmd

```diff
@@ -0,0 +1,13 @@
+
+echo on
+//set environment variables. Fill in the fields below, then run each line in the command window one at a time
+//then run the .NET Core app from the command window. You should be in the same folder as the arm-read-write.csproj file.
+CMD> dotnet run
+
+//Doing this instead of hardcoding these variables ensures that you don't accidentally save them to github.
+//
+SET IOT_DEVICE_ID=Contoso-Test-Device
```

Comment:
This file should be eliminated if we move to use a parameters file and CommandLineParser.